governance-with-multisig/README.md

```diff
@@ -0,0 +1,136 @@
+$ cd labs/governance-with-multisig
+$ npm install
+$ npx zos init governance-with-multisig
+```
+
+Our first step will be to register the `EthBox` contract in our project, deploy the logic contract to the blockchain, and create a new instance. We'll be working on a local ganache instance (you can start one via `./ganache-sh`), though you can also run this on any real network.
+
+```sh
+$ npx zos add EthBox
+$ npx zos push --skip-compile --network local
+$ npx zos create EthBox --network local
+```
+
+We can test that the EthBox instance was correctly deployed by `deposit`ing some ETH on it from a `truffle console`. Replace `MY_ETHBOX_ADDRESS` with the address returned by the last command run above.
+```js
+truffle(local)> EthBox.at(MY_ETHBOX_ADDRESS).deposit({ from: owner1, value: 1e18 })
+```
+
+Up to this point, the `admin` of the `EthBox` instance we have just created is a ZeppelinOS `App`, which centralizes control of our project. This App, in turn, is controlled by the account we used from the CLI. This means that this single account has absolute control to register new logic contracts, create new instances, and upgrade any of the existing ones.
+
+We will now create a multisig wallet, that will hold the keys to the upgreadeability of our new instance of `EthBox`. You can either create the wallet programatically from a `truffle console`, or use the Gnosis provided GUI, downloadeable from [here](https://github.com/gnosis/MultiSigWallet/releases). Take note of the multisig contract address.
+
+```js
+truffle(local)> MultiSigWallet.new([owner1, owner2, owner3], 2, { from: owner1 }).then(i => multisig = i)
+```
+
+To transfer ownership of the `EthBox` instance to the multisig, you can use the `change-admin` script provided here, by specifying the address of the multisig.
+
+```sh
+$ npx truffle exec change-admin.js EthBox $MULTISIG_ADDRESS --network local
+```
+
+Under the hood, the script is retrieving the `EthBox` instance address and the ZeppelinOS project `App` from the `zos.local.json` file, and using `zos-lib` to change the upgradeability admin of the instance:
+
+```js
+// Extract zOS App address and EthBox instance (proxy) address
+const networkInfo = JSON.parse(fs.readFileSync(`zos.${networkName}.json`));
+const appAddress = networkInfo.app.address;
+const proxyAddress = networkInfo.proxies[contractName][0].address;
+
+// Use the zos-lib App model and call changeProxyAdmin to set the new admin
+const app = await App.fetch(appAddress);
+await app.changeProxyAdmin(proxyAddress, newAdmin);
+```
+
+After this is run, the upgradeability admin of the `EthBox` instance is the multisig wallet itself. Nonetheless, keep in mind that **the owner of the ZeppelinOS App is still the original account**. This means that it is still possible to manage other contract instances (should you have created any), or deploy new logic contracts, from the CLI. Let's try this by registering `EthBoxV2` and deploying the logic contract to the blockchain.
```

Comment:
I would make clear that any other `EthBox` instance won't be owned by the multisig